pkg/daemon/daemon.go

```diff
@@ -180,6 +180,15 @@ func (dn *Daemon) getUnsupportedNicIdConfigMap() (*v1.ConfigMap, error) {
 }
 
 func (dn *Daemon) annotateUnsupportedNicIdConfigMap(cm *v1.ConfigMap, nodeName string) (*v1.ConfigMap, error) {
+	annotationKey := "openshift.io/" + nodeName
```

Comment:
Can you reuse "annotationKey" in line 200:
    cm.ObjectMeta.Annotations["openshift.io/"+nodeName] = string(jsonData)

Reply:
So, change line 200 to:
    cm.ObjectMeta.Annotations["openshift.io/"+nodeName] = string(jsonData)

Reply:
Done thanks.
Did more little refactors of annotateUnsupportedNicIdConfigMap